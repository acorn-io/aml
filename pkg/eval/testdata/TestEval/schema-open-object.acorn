schema {
    a: object
}

a: {
		b: 1
		c: 2
}
