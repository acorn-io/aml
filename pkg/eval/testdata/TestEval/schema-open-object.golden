{
  "Foo": {
    "type": "object",
    "properties": {
      "a": {
        "type": "object"
      }
    }
  },
  "a": {
    "b": 1,
    "c": 2
  }
}